arbos/programs/cache.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro/blob/master/LICENSE
+
+package programs
+
+import "github.com/ethereum/go-ethereum/common"
+
+type RecentPrograms struct {
```

Comment:
worth documenting somewhere that this is only inside transaction.

Reply:
Moved and documented in geth!